aten/src/ATen/native/native_functions.yaml

```diff
@@ -7146,63 +7146,63 @@
     CPU: foreach_tensor_div_list_kernel_slow_
     CUDA: foreach_tensor_div_list_kernel_cuda_
 
-- func: _foreach_add.ScalarList(Tensor[] tensors, float[] scalars) -> Tensor[]
+- func: _foreach_add.ScalarList(Tensor[] tensors, Scalar[] scalars) -> Tensor[]
```

Comment:
I have a question about the PythonArgParser. Given an integer list in python, like [0, 1, 2], which of the following do we parse it as?
- IntList
- FloatList
- ScalarList
Also, now that we have ScalarList, do we still need the codegen for float[] ? This might be a better question for the other diff in this stack

Reply:
Hmm, this will only be ambiguous if there are overloads of an operator that use some combination of the above (e.g., IntList overload, ScalarList overload)